tests/profile/serializers/test_metrics_serializer.py

```diff
@@ -35,9 +35,15 @@ def test_subindicator_not_none(profile_key_metric, geography):
     subindicator_data = subindicator(profile_key_metric, geography)
     assert subindicator_data != None
 
+    absolute_value_data = absolute_value(profile_key_metric, geography)
```

Comment:
let's split this up into two tests.

`test_subindicator_not_none` and `test_absolute_value_not_none` and can we add a test for `sibling` as well. I suspect there is a similar issue.